src/status_im/init/core.cljs

```diff
@@ -188,7 +188,7 @@
                         :accounts/accounts accounts
                         :mailserver/mailservers mailservers
                         :network-status network-status
-                        :network network
+                        :network account-network
```

Comment:
it probably should be `account-network-id` instead of `account-network`

Reply:
Not sure, I think it lacks a default value for new account, can you revert the commit?

Reply:
just checked previous commit 
<img width="237" alt="Screen Shot 2019-04-27 at 18 54 47" src="https://user-images.githubusercontent.com/2364994/56851984-f3659680-691d-11e9-9251-3fc9a666ed76.png">


I will file a fix in a moment